.travis.yml

```diff
@@ -1,28 +1,16 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
   - 5.5
   - 5.6
```

Comment:
During the implementation i allready tried it and i think it makes sence, but there are issues left. To not blow up the pull request, i think adding a seperate issue, as you proposed is good.